### PR TITLE
Code refactoring and support for claim format v0.1.0

### PR DESCRIPTION
**Pull request by edcdavid**
This is the inital code for the new repo. This PR includes the following:
- cleaned/refactored code, 
- separated HTML, Javascript and CSS code
-  adder linters for each component file 
-  support for "all-in-one" claim format v0.0.2. No need for classification.js anymore
- script to build a "results.html" bundle file and a github action to check that it is in sync with the source index.html

### CODE REVIEW

**Comment by sebrandon1** [comment]
Just some minor comments.

**Review by sebrandon1 — APPROVED** [review]

File: .github/workflows/checkbuild.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  check-result-html:
+    name: check if result.html is up to date with index.html
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (sebrandon1) [reply .github/workflows/checkbuild.yml:14]: Don't worry about it I'll update with #5

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  js-linter:
+    name: Javascript linter (standard)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (sebrandon1) [reply .github/workflows/linters.yml:14]: Dependabot will catch these with #5

File: .github/workflows/checkbuild.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      - name: install nodejs
+        run: sudo apt-get update; sudo apt-get -y install nodejs npm; npm install inline-source
+      - name: modify build sccript
+        run: sed -i 's/results.html/test.html/g' html/build.js
+      - name: generate test.html
+        run: cd html; node build.js
+      - name: compare results.html with test.html
+        run: cd html;diff results.html test.html
```
Comment (rdavid) [inline .github/workflows/checkbuild.yml:22]: Add the new line at the end.

File: .github/workflows/checkbuild.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    name: check if result.html is up to date with index.html
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: install nodejs
+        run: sudo apt-get update; sudo apt-get -y install nodejs npm; npm install inline-source
+      - name: modify build sccript
+        run: sed -i 's/results.html/test.html/g' html/build.js
+      - name: generate test.html
+        run: cd html; node build.js
```
Comment (rdavid) [inline .github/workflows/checkbuild.yml:20]:
```suggestion
        run: cd html && node build.js
```

File: .github/workflows/checkbuild.yml (hunk 3 above)
Comment (rdavid) [inline .github/workflows/checkbuild.yml:22]:
```suggestion
        run: cd html && diff results.html test.html
```

File: .github/workflows/checkbuild.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  workflow_dispatch:
+
+jobs:
+  check-result-html:
+    name: check if result.html is up to date with index.html
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: install nodejs
+        run: sudo apt-get update; sudo apt-get -y install nodejs npm; npm install inline-source
```
Comment (rdavid) [inline .github/workflows/checkbuild.yml:16]:
```suggestion
        run: sudo apt-get update && sudo apt-get -y install nodejs npm && npm install inline-source
```

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (rdavid) [inline .gitignore:1]: Add the new line at the end.

File: Makefile [hunk 7]
```diff
@@ -0,0 +1,2 @@
+build:
+	cd html;node build.js
```
Comment (rdavid) [inline Makefile:2]:
```suggestion
	cd html && node build.js
```

File: html/build.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+};
+
+inline.inlineSource(htmlFile, options)
+  .then((html) => {
+    fs.writeFileSync('results.html', html);
+    console.log('Build successful!');
+  })
+  .catch((err) => {
+    console.error('Error during build:', err);
+  });
```
Comment (rdavid) [inline html/build.js:16]: Add the new line at the end.

**Review by rdavid — APPROVED** [review]

File: .github/workflows/checkbuild.yml (hunk 3 above)
Comment (edcdavid) [reply .github/workflows/checkbuild.yml:22]: fixed

File: .github/workflows/checkbuild.yml (hunk 4 above)
Comment (edcdavid) [reply .github/workflows/checkbuild.yml:20]: fixed

File: .github/workflows/checkbuild.yml (hunk 3 above)
Comment (edcdavid) [reply .github/workflows/checkbuild.yml:22]: fixed

File: .github/workflows/checkbuild.yml (hunk 5 above)
Comment (edcdavid) [reply .github/workflows/checkbuild.yml:16]: fixed

File: .gitignore (hunk 6 above)
Comment (edcdavid) [reply .gitignore:1]: fixed

File: Makefile (hunk 7 above)
Comment (edcdavid) [reply Makefile:2]: fixed

File: html/build.js (hunk 8 above)
Comment (edcdavid) [reply html/build.js:16]: fixed

File: html/index.html [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+  </div>
+
+
+  <!-- This will load the initial JSON file, if present. It may fail. -->
+  <script src="./claimjson.js"></script>
+  <script src="./feedback.js"></script>
+
+</body>
+
+</html>
```
[the remark this answers is not in the document]
Comment (edcdavid) [reply html/index.html:163]: fixed

**Pull request merged**